adapter/internal/notifier/deployment_notifier.go

```diff
@@ -49,26 +50,34 @@ func SendRevisionUpdate(deployedRevisionList []*DeployedAPIRevision) {
 	}
 
 	jsonValue, _ := json.Marshal(deployedRevisionList)
-	req, _ := http.NewRequest("POST", revisionEP, bytes.NewBuffer(jsonValue))
+
 	// Setting authorization header
 	basicAuth := "Basic " + auth.GetBasicAuth(cpConfigs.Username, cpConfigs.Password)
-	req.Header.Set("Authorization", basicAuth)
-	req.Header.Set("Content-Type", "application/json")
+
+	logger.LoggerNotifier.Debugf("Revision deployed message sending to Control plane: %v", string(jsonValue))
 
 	// Adding 3 retries for revision update sending
 	retries := 0
 	for retries < 3 {
 		retries++
+
+		req, _ := http.NewRequest("PATCH", revisionEP, bytes.NewBuffer(jsonValue))
```

Comment:
Is the REST API resource is type patch to update the revision

Reply:
As a suggestion from @malinthaprasan, we changed the rest api type to patch.

Reply:
https://github.com/wso2/carbon-apimgt/pull/10698#discussion_r678804383